src/lua/debug.lua

```diff
@@ -0,0 +1,8 @@
+local debug = require('debug')
+
+local function sourcepath()
```

Comment:
It seems that names like `__file__`, `__dir__` are more common. Maybe also script_directory() would be more descriptive. Just thinking…

Reply:
Added little helpers in addition to `sourcepath` and `sourcedir` functions. Check those out

Reply:
Looks handy.